catalog/systemd.pl.catalog.in

```diff
@@ -353,3 +353,22 @@ Defined-By: systemd
 Support: %SUPPORT_URL%
 
 Jednostka @UNIT@ została ukończona, zużywając wskazane zasoby.
+
+-- 50876a9db00f4c40bde1a2ad381c3a1b
+Subject: System jest skonfigurowany w sposób, który może powodować problemy
+Defined-By: systemd
+Support: %SUPPORT_URL%
+
+Możliwe są następujące „etykiety”:
+• „split-usr” — /usr jest oddzielnym systemem plików, który nie został
+  zamontowany podczas uruchamiania systemd,
+• „cgroups-missing” — jądro zostało skompilowane bez obsługi cgroups
+  lub dostęp do oczekiwanych plików interfejsu jest ograniczony,
+• „var-run-bad” — /var/run nie jest dowiązaniem symbolicznym do /run,
+• „overflowuid-not-65534” — identyfikator użytkownika jądra używany
+  dla „nieznanych” użytkowników (za pomocą przestrzeni nazw użytkowników
+  lub NFS) nie wynosi 65534,
```

Comment:
identyfikator użytkownika dla „nieznanych” użytkowników (przy wykorzystaniu przestrzeni nazw użytkowników lub NFS) ... ?

Reply:
The original has the word “kernel”, should I skip it in translation?

Reply:
I’ve changed it to skip “kernel” for now. Let me know if it’s ok.